examples/hallowelt/go_app/README.md

```diff
@@ -0,0 +1,253 @@
+# HelloWorld tutorial using Go client Fabric SDK
+This tutorial shows how to invoke Fabric Private Chaincode using the Go Fabric Client SDK and the Fabric-Private-chaincode(FPC) framework.  This assumes familiarity with the [concepts](https://hyperledger-fabric.readthedocs.io/en/release-2.0/whatis.html#), [the programming model](https://hyperledger-fabric.readthedocs.io/en/release-2.0/chaincode.html) and the [gateway] (https://hyperledger-fabric.readthedocs.io/en/release-2.0/developapps/gateway.html) in Hyperledger Fabric v2.0.
+
+## Prerequisites
```

Comment:
I guess this subsection is not needed when this entire readme got integrated into the hello world tutorial as subsection.

Reply:
yes probably

Reply:
ok, remove prerequisites.